pandas/util/testing.py

```diff
@@ -780,17 +780,17 @@ def assert_is_valid_plot_return_object(objs):
     if isinstance(objs, (pd.Series, np.ndarray)):
         for el in objs.ravel():
             msg = (
-                "one of 'objs' is not a matplotlib Axes instance, type "
-                "encountered {name!r}"
-            ).format(name=el.__class__.__name__)
+                "one of 'objs' is not a matplotlib Axes instance, "
+                f"type encountered {repr(type(el).__name__)}"
+            )
             assert isinstance(el, (plt.Axes, dict)), msg
     else:
-        assert isinstance(objs, (plt.Artist, tuple, dict)), (
-            "objs is neither an ndarray of Artist instances nor a "
-            'single Artist instance, tuple, or dict, "objs" is a {name!r}'.format(
-                name=objs.__class__.__name__
-            )
+        msg = (
+            "objs is neither an ndarray of Artist instances nor a single "
+            'Artist instance, tuple, or dict, "objs" is a '
```

Comment:
Think this should be in double quotes with 'objs' in single quotes to be consistent with line 783